state-chain/runtime/src/chainflip/bitcoin_elections.rs

```diff
@@ -523,3 +526,63 @@ pub fn initial_state() -> InitialStateOf<Runtime, BitcoinInstance> {
 		shared_data_reference_lifetime: 8,
 	}
 }
+
+pub struct BitcoinGovernanceElectionHook;
+
+#[derive(Clone, PartialEq, Eq, Debug, Encode, Decode, TypeInfo)]
+pub enum ElectionTypes {
+	DepositChannels(ElectionPropertiesDepositChannel),
+	Vaults(ElectionPropertiesVaultDeposit),
+	Egresses(ElectionPropertiesEgressWitnessing),
+}
+
+impl pallet_cf_elections::GovernanceElectionHook for BitcoinGovernanceElectionHook {
+	type Properties = (<BitcoinChain as ChainTypes>::ChainBlockNumber, ElectionTypes);
+
+	fn start(properties: Self::Properties) {
+		match properties {
```

Comment:
nit, could `if let Err()` here and log error - though will only be corrupt storage error, we can at least log the properties alongside it